src/pudl/transform/ferc1.py

```diff
@@ -1173,18 +1201,19 @@ def calculate_values_from_components(
             pd.merge(
                 calculation_components,
                 data,
-                validate="one_to_many",
+                validate=calc_to_data_merge_validation,
```

Comment:
we were not able to test the `depreciation_by_function` table because it has two dimension columns and child calc components *necessarily & correctly* are part of more than one parent calculation.